schema.go

```diff
@@ -149,17 +149,20 @@ func StreamingGetOldBuilds(ctx context.Context) (<-chan LogKeeperBuild, <-chan e
 			return
 		}
 
-		for cur.Next(ctx) {
+		for cur.Next(db.Context()) {
 			build := LogKeeperBuild{}
 			if err := cur.Decode(&build); err != nil {
 				errOut <- err
 				return
 			}
 			out <- build
-			build = LogKeeperBuild{}
+
+			if ctx.Err() != nil {
```

Comment:
Shouldn't this check the DB context?

Reply:
Calls to `Next` implicitly check the `db.Context()` so the for loop will exit if the application context is cancelled.
This check is meant to accomplish the thing I described in the previous comment.